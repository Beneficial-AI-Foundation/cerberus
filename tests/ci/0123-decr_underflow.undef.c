#include <limits.h>
int main(void)
{
  int x = INT_MIN;
  signed char = SCHAR_MIN;
  c--; // defined because of the promotion
  x--;
}
