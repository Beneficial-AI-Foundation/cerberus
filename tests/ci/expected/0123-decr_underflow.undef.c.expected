Undefined {ub: "UB036_exceptional_condition", stderr: "", loc: "<5:3--5:6>"}
